<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketMeshEditorWidget</class>
 <widget class="QWidget" name="RocketMeshEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>448</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Containers */

QWidget#RocketMeshEditorWidget {
	background-color: black;
	border: 0px;
}

QLabel#renderTargetWidget {
	background-color: black;
	border: 0px;
}

QFrame#frameControls {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgb(232, 234, 233), stop:1 rgb(245, 246, 247));
	border: 0px;
	border-right: 1px solid gray;
}

/* Color buttons */

QPushButton#buttonAmbient, #buttonDirectional, #buttonBackground {
	min-height: 23px;
	max-height: 23px;
	min-width: 45px;
	border-radius: 0px;
	background-color: transparent;
	font-size: 10t;
	border: 1px solid grey;
	color: rgb(100,100,100);
}

QPushButton#buttonAmbient:hover, #buttonDirectional:hover, #buttonBackground:hover {
	background-color: transparent;
}

/* Spesific styles */

QLabel#rotationTitle, #lightsTitle, #materialsTitle {
	font-size: 17px;
	font-weight: bold;
	color: rgb(243,154,41);
	padding: 0px;
}

/* Generic styles */

QSpinBox, QDoubleSpinBox {
	min-height: 20px;
	max-height: 20px;
}

QLabel, QCheckBox, QRadioButton {
	font-family: &quot;Arial&quot;;
	font-size: 13px;
	color: rgb(100,100,100);
	font-weight: bold;
}

QCheckBox:hover {
	color: rgb(50,50,50);
}

QCheckBox::indicator:unchecked {
	image: url(:/images/icon-unchecked.png);
}

QCheckBox::indicator:checked {
	image: url(:/images/icon-checked.png);
}

QComboBox {
	border: 0px;
	border-bottom: 1px solid grey;
	font-family: &quot;Arial&quot;;
	font-size: 12px;
	color: rgb(65, 65, 65);
	font-weight: bold;
	padding-left: 0px;
    min-width: 90px;
	min-height: 20px;
	max-height: 20px;
	background-color: transparent;
}

QComboBox:hover {
	border-bottom: 1px solid black;
}

QComboBox:disabled, :disabled:hover {
	background-color: rgba(100,100,100,100);
}

QComboBox:down-arrow {
	image: url(':/images/icon-combobox-down-arrow.png');
	width:12px;
}

QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: center right;
	padding-right: 2px;
	width: 16px;
	max-height: 11px;
	border: 0px;
}

QComboBox QAbstractItemView {
	border: 1px solid grey;	
	background-color: rgb(255, 255, 255);
	selection-background-color: rgba(230,230,230,120);
	selection-color: black;
	padding: 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="mainHorizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="renderTargetWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameControls">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="rotationTitle">
          <property name="text">
           <string>Rotation</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="rotationGridLayout">
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="spacing">
           <number>8</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="rotationXLabel">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="rotationXSlider">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximum">
             <number>359</number>
            </property>
            <property name="pageStep">
             <number>45</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>90</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="rotationYLabel">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="rotationYSlider">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximum">
             <number>359</number>
            </property>
            <property name="pageStep">
             <number>45</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>90</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="rotationZLabel">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="rotationZSlider">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximum">
             <number>359</number>
            </property>
            <property name="pageStep">
             <number>45</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>90</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="autoRotateX">
            <property name="maximumSize">
             <size>
              <width>18</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Auto rotate axis</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="autoRotateY">
            <property name="maximumSize">
             <size>
              <width>18</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Auto rotate axis</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="autoRotateZ">
            <property name="maximumSize">
             <size>
              <width>18</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Auto rotate axis</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lightsTitle">
          <property name="styleSheet">
           <string notr="true">margin-top: 10px;</string>
          </property>
          <property name="text">
           <string>Rendering</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="spacing">
           <number>8</number>
          </property>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonAmbient">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="buttonDirectional">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="directionalCheckBox">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Directional</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="backgroundLabel">
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttonBackground">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="ambientLabel">
            <property name="text">
             <string>Ambient</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QCheckBox" name="boundingBoxCheckBox">
            <property name="text">
             <string>Bounding box</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="wireframeCheckBox">
            <property name="text">
             <string>Wireframe</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="materialsTitle">
          <property name="styleSheet">
           <string notr="true">margin-top: 10px;</string>
          </property>
          <property name="text">
           <string>Materials</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="materialsLayout"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="statsTextEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: -4px; border: 0px; background-color: transparent;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RocketPlugin.qrc"/>
 </resources>
 <connections/>
</ui>
